code/style.css

```diff
@@ -1,29 +1,96 @@
 * {
   box-sizing: border-box;
+  margin: 0;
+  padding: 0;
 }
 
 body {
   margin: 0;
```

Comment:
You already have body and margin set to 0 on row 1 so maybe it's not necessary to do it here twice? 😃